editor/inspector/components/label.js

```diff
@@ -0,0 +1,204 @@
+                        image.setAttribute('value', `packages://scene/static/icons/align-v-${iconName}.png`);
+                    }
+                    image.style.height = '20px';
+                    image.style.width = '22px';
+                    image.style.verticalAlign = 'middle';
+                    image.setAttribute('fill', true);
+                    image.setAttribute('tooltip', dump.tooltip);
+                    image.setAttribute('readonly', true);
+                    button.appendChild(image);
+                    content.appendChild(button);
+                }
+                prop.appendChild(label);
+                prop.appendChild(content);
+                return prop;
+            },
+            update(element, dump) {
+                const tab = element.querySelector('ui-tab');
+                if (isMultipleInvalid(dump.verticalAlign)) {
+                    tab.value = -1;
+                } else {
+                    tab.value = dump.verticalAlign.value;
+                }
+            },
+        },
+        isBold: {
+            create() {
+                const prop = document.createElement('ui-prop');
+                const label = document.createElement('ui-label');
+                label.setAttribute('slot', 'label');
+                label.value = 'FontStyle';
+                prop.appendChild(label);
+                const content = document.createElement('div');
+                content.setAttribute('slot', 'content');
+                content.classList.add('fontStyleParent');
+                const styles = ['isBold', 'isItalic', 'isUnderline'];
+                const styleDisplayNames = ['B', 'I', 'U'];
+                for (let index = 0; index < styles.length; index++) {
+                    const style = styles[index];
+                    const div = document.createElement('div');
+                    div.innerHTML = styleDisplayNames[index];
+                    div.setAttribute('key', style);
+                    div.classList.add('fontStyle');
+                    div.addEventListener('mouseup', () => {
+                        prop.dump = this.dump.value[style];
+                        prop.dump.value = !prop.dump.value;
+                        if (prop.dump.values) {
+                            prop.dump.values = prop.dump.values.map(() => prop.dump.value);
+                        }
+                        prop.dispatch('change-dump');
+                        this.dump.value[style].value ? div.classList.add('select') : div.classList.remove('select');
+                    });
+                    content.appendChild(div);
+                }
+                prop.appendChild(content);
+                return prop;
+            },
+            update(element, dump) {
+                const parent = element.querySelector('[slot="content"]');
+                for (let index = 0; index < parent.childNodes.length; index++) {
+                    const div = parent.childNodes[index];
```

Comment:
depend on `const styles = ['isBold', 'isItalic', 'isUnderline'];` to loop select div element ,would be more stable.